src/domain/collaboration/callout-framing/callout.framing.service.ts

```
@@ -132,39 +175,85 @@ export class CalloutFramingService {
+            calloutFraming,
+            {
+              profile: {
+                displayName: 'Callout Framing Whiteboard',
+              },
+              content: calloutFramingData.whiteboardContent,
+            },
+            storageAggregator,
+            userID
+          );
+        }
       }
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)